landscape/explanation/glossary/distribution.md

```diff
@@ -0,0 +1,23 @@
+# Distribution
+
+In Landscape, a distribution refers to a specific release of the Ubuntu operating System. You can reference it with either the code name (for example jammy), the version number such as 22.04.4, or the actual name (Ubuntu) depending on your current use case.
```

Comment:
```suggestion
In Landscape, a **distribution** refers to the Ubuntu operating system. There are different ways to refer to the distribution depending on your use case.
```

Reply:
Just some rewording. You did a really great job with your research on distributions in Landscape!!!! I wanted to keep this part more general though, since there are some inconsistencies with how the Landscape API refers to distributions (e.g., sometimes it's the version number and sometimes it's the name). Your information isn't necessarily incorrect, though - from a documentation perspective, I just think it'd be better to provide a general explanation here, and then users can dig into their specific use case for those additional details. Let me know if you have any questions on that :smiley:

Reply:
Right!! I think this is a better approach too. Something for me to consider for future documentation. Thank you!